x-pack/plugins/alerting/server/lib/is_rule_snoozed.test.ts

```diff
@@ -299,7 +299,7 @@ describe('isRuleSnoozed', () => {
       },
     ];
 
-    expect(isRuleSnoozed({ snoozeSchedule: snoozeScheduleA, muteAll: false })).toBe(false);
+    expect(isRuleSnoozed({ snoozeSchedule: snoozeScheduleA, muteAll: false })).toBe(true);
```

Comment:
This test is, and always was, expected to return `false`. It is supposed to begin at 00:00 in Taipei, every Wednesday, and have a `duration` of **one minute**. If `Date.now()` is 00:00 GMT, the snooze should be `false`.

Changing the `dtstart` to midnight in Taipei and converting it to GMT (the fix applied in https://github.com/elastic/kibana/pull/152873/files#diff-5c294b91450cdaf646e10d6425e4d1c7a0d4ffd5b1502022d23be521f99719d3R18) also causes this test to fail in this new PR.

Reply:
So for this test, after staring at it for a while, I think it was initialized incorrectly (If the goal was to test 00:00 Taipei). Since we're setting `dtstart = DATE_2019,`, which is `2019-01-01T00:00:00.000Z UTC`.

Because we are using UTC for all dates in our application (dates sent back from the frontend are all in UTC regardless of the client's timezone), If we wanted it to test 00:00 Taipei time, then the dtstart would need to be `2018-12-31T16:00:00.000Z UTC`, if that makes sense.

Reply:
Right, the `dtstart` for this test should be `2018-12-31T16:00:00.000Z` and it should be expected to be `false`.

If you can define `const DATE_2019_IN_ASIA = '2018-12-31T16:00:00.000Z';` up top and then reset this test to expect `false`, it now works in the latest commit